src/KDSoapClient/KDSoapClientInterface.h

```diff
@@ -346,6 +346,23 @@ class KDSOAP_EXPORT KDSoapClientInterface
       */
     void setTimeout(int msecs);
 
+    /**
+     * @brief setUseWsAddressing
+     * @param useWsAddressing
+     * If this parameter is set to true then the outgoing messages
+     * SOAP action parameter will be added as an Action field of the
+     * SOAP envelope instead of passing it as URL parameter.
+     */
+    void setUseWsAddressing(bool useWsAddressing);
```

Comment:
I think you should metion WS-Addressing and HTTP headers. And maybe mention the specifications that define it.

Reply:
Fixed.